CHANGELOG.md

```diff
@@ -1,601 +1,179 @@
+  introduced to follow Debian policy. For backward compatibility, if 
+  you upgraded from v4, `td-agent` user/group remains as same
+  `UID`/`GID` of `_fluentd`. This change makes easy to
+  keep using `/etc/logrotate.d/td-agent` as is.  (#475,#519)
+  * Note that process or file owner of `fluent-package` is displayed
+    as `td-agent` instead of `fluentd`.
+* deb: the path of service configuration file is changed to
+  `/etc/default/fluentd`. (#461)
+* deb: `fluentd-apt-source` was renamed to `fluent-apt-source`.
+  * You can remove transitional `fluentd-apt-source` after upgrading
+    to `fluent-apt-source`. (#507,#514,#515)
+
+#### For RHEL user:
+
+* rpm: the service file is changed to `fluentd.service`.
+  It provides `td-agent` as an alias. Note that if you
+  want to keep using `td-agent` as a service name, you must
+  explicitly execute the following commands: (#461,#516)
+  
+  ```
+  $ sudo systemctl enable fluentd
+  ```
+
+* rpm: user/group name was changed to fluentd. For backward
+  compatibility, if you upgraded from v4, `td-agent` user/group
+  remains as same `UID`/`GID` of `fluentd`. This change makes easy to
+  keep using `/etc/logrotate.d/td-agent` as is. (#475,#519)
+  * Note that process or file owner of `fluent-package` is displayed
+    as `td-agent` instead of `fluentd`.
+* rpm: the path of service configuration file is changed to
+  `/etc/sysconfig/fluentd`. (#461)
+* rpm: prelink configuration was removed. (#472,#529)
+  * In recent days, it is common to disable prelink configuration. If
+  you upgrade from v4, that configuration file
+  (`/etc/sysconfig/prelink.conf.d/td-agent-ruby.conf`) itself will be
+  removed or the entry about `td-agent` will be removed from
+  `/etc/sysconfig/prelink.conf`.
+* rpm: added support for Amazon Linux 2023. (#459)
+
+#### For Windows user:
+
+* msi: renamed to "Fluent Package" (#463,#466,#471)
+* msi: the default install path of `fluent-package` was changed to `c:/opt/fluent`.
+  Note that old log files are kept as is. The following files are migrated to the
+  new path: (#466,#469,#487)
+  
+  * `c:/opt/td-agent/etc/td-agent/td-agent.conf`
+  * `c:/opt/td-agent/etc/plugins/*`
+* msi: the prefix of batch files were renamed to `fluent*`.
+  Thus `td-agent-prompt.bat` was renamed to `fluent-package-prompt.bat`. (#484)
+* msi: disable auto starting service after install. (#521)
+
+  * If you want to start `fluentd` as a service, execute the following command with administrator privileges.
+  
+  ```
+  > net start fluentdwinsvc
+  ```
+
+* msi: stop customizing icons for file browser (#469)
+* msi: update resources for `fluent-package` (#470)
+* msi: changed the default path of buffer/failed_records (#527)
+
+#### For macOS user:
+
+* dmg: renamed to `fluent-package` (#474,#478,#479,#480,#481,#482,#483)
+  * Note that the .dmg package support will be dropped in the future
+  release. We plans to migrate for `homebrew` ecosystem.
```

Comment:
We have currently no plan to release dmg version of fluent-package v5 and not tested enough, so we should notify it at first.

Reply:
Thanks, I've added it.